python/tank/bootstrap/configuration_writer.py

```diff
@@ -473,7 +482,8 @@ def write_pipeline_config_file(self, pipeline_config_id, project_id, plugin_id,
             "published_file_entity_type": "PublishedFile",
             "use_bundle_cache": True,
             "bundle_cache_fallback_roots": bundle_cache_fallback_paths,
-            "use_shotgun_path_cache": True
+            "use_shotgun_path_cache": True,
+            "descriptor": descriptor.get_dict()
```

Comment:
Not sure, but wondering if it's clearer to call this `source_descriptor` or something like that? Meaning that this is not a descriptor pointing at where the config is right now but rather a descriptor explaining where the config came from?